__tests__/app.test.js

```diff
@@ -155,14 +171,6 @@ describe("GET /api/articles", () => {
         expect(body.msg).toBe("not found");
       });
   });
-  test("404: Responds with an appropriate status and error message when given a valid but non-existent topic", () => {
-    return request(app)
-      .get("/api/articles?topic=non-existant-topic")
-      .expect(404)
-      .then(({ body }) => {
-        expect(body.msg).toBe("not found");
-      });
-  });
```

Comment:
I am not exactly sure what happened with this branch but not all of your code seems to have been added correcty?
 
 But from looking at the rest of your tests you are still missing a test:

You should have a test to assert that multiple queries can work together:
E.g `/api/articles?order=asc&sort_by=title`